hack/demo.sh

```diff
@@ -48,10 +48,11 @@ spec:
   configuration:
     exclude:
     - not_useful
```

Comment:
We removed `not_useful` (see https://github.com/hacbs-contract/ec-policies/pull/236), perhaps it's a good opportunity to remove it here as well